### PR TITLE
Update docs, default track positions, higlass-version

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

- Small docs update
- Add tracks without `horizontal-` to default positions
- Bump higlass version

Unfortunately some automatic formatting changes snuck through as well.

Why is it necessary?

Fixes #___

## Checklist

- [ ] Unit tests added or updated
- [x] Documentation added or updated
- [x] Updated CHANGELOG.md
- [ ] Ran `black` on the root directory

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [bumpversion]
-current_version = 0.4.0
+current_version = 0.4.4
```
Comment (pkerpedjiev) [inline .bumpversion.cfg:2]: This is because the previous releases hadn't used bumpversion so we're skipping ahead a bit.

File: higlass/client.py [hunk 2]
```diff
@@ -143,9 +150,7 @@ def change_attributes(self, **kwargs):
         """
         conf = self.conf.copy()
         conf.update(kwargs)
-        return self.__class__(
-            conf["type"], position=self.position, tileset=self.tileset, **conf
-        )
+        return self.__class__(conf["type"], position=self.position, tileset=self.tileset, **conf)
```
Comment (pkerpedjiev) [inline higlass/client.py:153]: Formatting 😐

File: higlass/client.py [hunk 3]
```diff
@@ -209,12 +214,8 @@ def __init__(self, numerator, denominator, *args, **kwargs):
                 f"Different track types: {numerator.conf['type']}, {denominator.conf['type']}"
             )
 
-        if json.dumps(numerator.conf["options"]) != json.dumps(
-            denominator.conf["options"]
-        ):
-            logger.warn(
-                "Tracks have different options, so we're using the first track's"
-            )
+        if json.dumps(numerator.conf["options"]) != json.dumps(denominator.conf["options"]):
```
Comment (pkerpedjiev) [inline higlass/client.py:217]: Formatting 😐

File: higlass/client.py [hunk 4]
```diff
@@ -462,9 +463,7 @@ def from_dict(cls, conf):
                 # position has to be passed in as part of the parameter
                 # array so that the constructor can be called with it as
                 # a parameter
-                self.add_track(
-                    track=klass.from_dict({"position": position, **track_conf})
-                )
+                self.add_track(track=klass.from_dict({"position": position, **track_conf}))
```
Comment (pkerpedjiev) [inline higlass/client.py:466]: Formatting 😐

File: higlass/client.py [hunk 5]
```diff
@@ -527,9 +526,7 @@ def add_chrominfo(self, track):
 class ViewConf(Component):
     """Configure a dashboard"""
 
-    def __init__(
-        self, views=[], location_syncs=[], value_scale_syncs=[], zoom_syncs=[]
-    ):
+    def __init__(self, views=[], location_syncs=[], value_scale_syncs=[], zoom_syncs=[]):
```
Comment (pkerpedjiev) [inline higlass/client.py:529]: Formatting 😐

File: higlass/client.py [hunk 6]
```diff
@@ -680,9 +677,7 @@ def from_link(cls, url):
         else:
             raise ValueError("Not a valid viewconf server")
 
-        endpoint = urlunsplit(
-            (parts.scheme, parts.netloc, "api/v1/viewconfs", "d=" + conf_id, "")
-        )
+        endpoint = urlunsplit((parts.scheme, parts.netloc, "api/v1/viewconfs", "d=" + conf_id, ""))
```
Comment (pkerpedjiev) [inline higlass/client.py:680]: Formatting 😐

File: .bumpversion.cfg (hunk 1 above)
Comment (flekschas) [reply .bumpversion.cfg:2]: Fast forward into a brighter and better future! 🚀 May bumpversion be with us!

File: higlass/client.py (hunk 2 above)
Comment (flekschas) [reply higlass/client.py:153]: Is this personal flavor or does it come from black?

File: higlass/client.py [hunk 7]
```diff
@@ -1,9 +1,9 @@
 from copy import deepcopy
 import json
-import slugid
 import logging
 import os
 
+import slugid
```
Comment (flekschas) [inline higlass/client.py:6]: What is the point of this change?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,10 @@
-## Next
+##
 
-- Add constructor for multivec tilesets
+## [v0.4.4](https://github.com/higlass/higlass-python/compare/v0.4.4...v0.4.3)
+
+- Added default positions for non `horizontal-` tracks
+- Update docs to include Views import in simple example
+- Bump higlass version
```
Comment (flekschas) [inline CHANGELOG.md:7]:
```suggestion
- Bump HiGlass to v1.11
```

**Review by flekschas — APPROVED** [review]
Sweet!

File: higlass/client.py (hunk 2 above)
Comment (pkerpedjiev) [reply higlass/client.py:153]: Black.

**Pull request merged**